lib/linux/background-managers/pcmanfm.js

```diff
@@ -0,0 +1,9 @@
+'use strict';
+
+const {commandExists, execFile} = require('../util');
+
+exports.isAvailable = () => commandExists('pcmanfm');
+
+exports.set = async imagePath => {
+	await execFile('pcmanfm', ['-w ' + imagePath]);
```

Comment:
Use the long-flag and JS template literal.

https://www.mankier.com/1/pcmanfm#--set-wallpaper